nextflow.config

```
@@ -214,10 +240,13 @@ manifest {
     homePage        = 'https://github.com/nf-core/nanoseq'
     description     = 'A pipeline to demultiplex, QC and map Nanopore data'
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=21.04.0'
+    nextflowVersion = '!>=21.10.3'
     version         = '2.0.1'
```

```suggestion
    version         = '3.0.0'
```
Was `nf-core bump version 3.0.0` run?